bundles/org.eclipse.equinox.p2.core/META-INF/MANIFEST.MF

```diff
@@ -63,15 +63,14 @@ Export-Package: org.eclipse.equinox.internal.p2.core;x-friends:="org.eclipse.equ
    org.eclipse.equinox.p2.updatesite,
    org.eclipse.equinox.p2.director.app,
    org.eclipse.equinox.p2.transport.ecf",
- org.eclipse.equinox.p2.core;version="2.13.100";uses:="org.eclipse.core.runtime",
+ org.eclipse.equinox.p2.core;version="2.13.200";uses:="org.eclipse.core.runtime",
```

Comment:
Need to investigate why package version is incremented here, this seem to be wrong, it is likely that Tycho just do it because it matches the project version.